src/User32.Desktop/User32+DISPLAYCONFIG_SOURCE_MODE_INFO.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DISPLAYCONFIG_SOURCE_MODE_INFO"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct DISPLAYCONFIG_SOURCE_MODE_INFO
+        {
+            public uint bitvector1;
```

Comment:
@AArnott  Makes sense.
@vbfox , this is the second part of a union. One with a very strange definition by the way ;-) I didn't like this `DUMMYSTRUCTNAME`, and changed it to something more meaningful as well as the user will not access it directly. Looked like a lame name for me (heheh).

```
struct {
      UINT32 cloneGroupId  :16;
      UINT32 sourceModeInfoIdx  :16;
    } DUMMYSTRUCTNAME;
```

> **DUMMYSTRUCTNAME**
>
> **cloneGroupId**
> A valid identifier used to show which clone group the path is a member of only when `DISPLAYCONFIG_PATH_SUPPORT_VIRTUAL_MODE` is set. If this value is invalid, then it must be set to `DISPLAYCONFIG_PATH_CLONE_GROUP_INVALID.`
`cloneGroupId` is only used when the source mode index is not specified. Two such scenarios are when the source mode info must be invalid because `SDC_TOPOLOGY_SUPPLIED` is used, and when `SDC_USE_SUPPLIED_DISPLAY_CONFIG` is used with paths that do not have source mode info. The `cloneGroupId` will be used to indicate which paths are in a clone group, all the paths with the same `cloneGroupId` value are considered in the same clone group. There is no requirement that the clone group id’s have to be zero based or contiguous. Supported starting in Windows 10.
>
> **sourceModeInfoIdx**
> A valid index into the mode array of the `DISPLAYCONFIG_SOURCE_MODE` entry that contains the source mode information for this path only when `DISPLAYCONFIG_PATH_SUPPORT_VIRTUAL_MODE` is set. If there is no entry for this in the mode array, the value of `sourceModeInfoIdx` is `DISPLAYCONFIG_PATH_SOURCE_MODE_IDX_INVALID`. Supported starting in Windows 10.